03debianWheezy_tar/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM debian:wheezy
+
+MAINTAINER Eugene Naryshkin aurcame@gmail.com
+
+RUN apt-get update && apt-get install -y \
+      wget \
+      build-essential
+
+RUN wget http://download.redis.io/redis-stable.tar.gz
+
+RUN tar xvzf redis-stable.tar.gz
```

Comment:
This is not good approach as 1) 'wget' command is not recessary to be available at the container runtime, 2) tar archive will be added into a layer. Better solution would be to combine all these RUN commands into a single chain (command).